src/main/modules/nunjucks/index.ts

```diff
@@ -126,6 +126,8 @@ export class Nunjucks {
     nunjucksEnv.addGlobal('HowMuchPaidClaimantOption', HowMuchPaidClaimantOption)
     nunjucksEnv.addGlobal('Service', Service)
     nunjucksEnv.addGlobal('cookieText', `GOV.UK uses cookies make the site simpler. <a href="${AppPaths.cookiesPage.uri}">Find out more about cookies</a>`)
+    nunjucksEnv.addGlobal('moneyClaims', `Money Claims`)
+    nunjucksEnv.addGlobal('headingVisible', `true`)
```

Comment:
the true shouldn't be wrapped in quotes

Reply:
But the addGlobal function expects two strings

Reply:
according to the method the second option is of type any: 
```addGlobal(name: string, value: any)```
and using true is more accurate from a type point of view

Reply:
Ok, I'll change and see if the application still runs ok. I thought it was two strings but yes your correct its a string and any